cleancat/base.py

```diff
@@ -491,12 +493,31 @@ class Enum(Choices):
     """
     Like Choices, but expects a Python 3 Enum.
     """
+    def __init__(self, choices, **kwargs):
+        """Initialize the Enum field.
+
+        The `choices` param can be either:
+        * an enum.Enum class (in which case all of its values will become
+          valid choices),
+        * a list containing a subset of the enum's choices (e.g.
+          `[SomeEnumCls.OptionA, SomeEnumCls.OptionB]`). You should provide
+          more than one choice in this list and *all* of the choices should
+          belong to the same enum class.
```

Comment:
I like to use "must" or "need to" since it's stronger (you used "should" twice here).